### ISSUE TITLE
Bug? UI in Dockerimage not able to load all resources

### ISSUE THREAD

**Issue by VfBfoerst** [Portkey-AI/gateway#829]
### What Happened?

Hi :)  
I wanted to try the docker image (I tried version 1.8.5, 1.8.4, 1.8.3), but the ui does not show up as expected. It fails to load the additional ressources.  
UI looks at the moment like that:   
![image](https://github.com/user-attachments/assets/171835b3-b9da-497a-a376-67fd2c82d361)
  
Console: 
![image](https://github.com/user-attachments/assets/33b046f5-7ead-44ce-a4c0-ff713d4832bb)
  
As copy-paste:  
```bash  
Failed to load resource: the server responded with a status of 404 (Not Found)
tabs.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
buttons.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
interative-code.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
modal.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
styles/style.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
logs.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
snippets.js:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
main.js:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
hotjar-3502316.js?sv=7:3  For security reasons, Hotjar only works over HTTPS. Learn more: https://help.hotjar.com/hc/en-us/articles/115011624047
window.hjBootstrap @ hotjar-3502316.js?sv=7:3
wtp.js:17 handleResponse
wtp.js:20 file_name is 
wtp.js:22 path is chrome-extension://dlmffppdphadfmicefckcdnpaokpdmbm/
wtp.js:24 type of tmext is undefined
wtp.js:25 type of message is undefined
public:1  Error handling response: TypeError: Cannot read properties of undefined (reading 'databc')
    at handleResponse (chrome-extension://dlmffppdphadfmicefckcdnpaokpdmbm/wtp.js:26:46)
    at chrome-extension://dlmffppdphadfmicefckcdnpaokpdmbm/wtp.js:83:5
public:1  Unchecked runtime.lastError: A listener indicated an asynchronous response by returning true, but the message channel closed before a response was received
[NEU] Erläutern von Konsolenfehlern mithilfe von Copilot in Edge: Klicken Sie auf
         , um einen Fehler zu erklären.
        Weitere Informationen
        Nicht mehr anzeigen
Microsoft Edge is moving towards a new experience that allows users to choose to browse without third-party cookies.
tabs.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
header.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
interative-code.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
buttons.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
logs.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)
modal.css:1       
        Failed to load resource: the server responded with a status of 404 (Not Found)
style.css:1 
        Failed to load resource: the server responded with a status of 404 (Not Found)

```

I tried edge browser as well as firefox.   

### What Should Have Happened?

UI should be able to load all of its resources succesfully. 

### Relevant Code Snippet

Deployment via:  
```bash
docker run --rm  -p 8787:8787 portkeyai/gateway:1.8.5
```

**Comment by VisargD** [comment]
Hey, we are looking into this issue. Thanks for reporting!

**Comment by roh26it** [comment]
@VfBfoerst Can you try this url once - "http://localhost:8787/public/"

The last slash might be what's causing the issue. Will fix this in the next release.

**Comment by VfBfoerst** [comment]
@roh26it thank you! Yeah i will try it tomorrow.

**Comment by VfBfoerst** [comment]
Can confirm its because of the slash. Works with the slash at the end.

**Comment by VisargD** [comment]
We have also pushed a fix for this in #832. It will be a part of next release happening today

**Comment by VisargD** [comment]
This went live in v1.8.6.
